Translations/translation_EN.json

```diff
@@ -297,14 +297,14 @@
 				"Power pulse",
 				"delay"
 			],
-			"desc": "Delay before keep-awake pulse is triggered (x 2.5s)"
+			"desc": "Delay before keep-awake pulse is triggered <x 2.5s>"

```

Comment:
Convention I know of here is to use `()` rather than `<>` but also not really bothered. Does anyone have an opinion?

Reply:
Good morning :)

I was referring to them as a whole. I'm also a bit "eh" and not worried really, so long as they all match I'm happier.

Truth told, I rarely look at the descriptions so I hadn't noticed the trend to <>.

Happy to merge as is, or go to (), your preference is fine by me here :)

Reply:
Opinion:

< > should be used then documenting combobox like selections.

( ) should be used then there is optional not necessary information about the setting like guidelines, recommendations.

User should not do extra math to know what value really means. However maybe in this case [ ] should be used to determine stepping coefficient and snippet could be placed to description beginning, instead of the end.

Anyways there should be ruling consistence in whole file.

Reply:
@Ralim
Sorry, but I did not want to open an extra issue on the following:
Could you please clarify, what this setting is meant for?

**`"desc": "Power source. Sets cutoff voltage. <DC 10V> <S 3.3V per cell, disable power limit>"`**

Especially the **`disable power limit`** is confusing.
Or am I missing something here?

Thank you in advance.